frontend/components/NctrlValueWidgets.jsx

```diff
@@ -143,13 +155,13 @@ export function NctrlValueSlider({ path, options, min, max, rerender, rerenderDe
     } else if (value < min) {
       return <ArrowLeft />;
     } else {
-      return value.toFixed(value > 100 ? 0 : 1);
+      return value.toFixed(integer || value > 100 ? 0 : 1);
```

Comment:
what does this do?

Reply:
Ensures that label shows integer values when the slider options integer is enabled.

Reply:
i see